### PR TITLE
Update requirements.txt remove mypy

### PR DESCRIPTION
**Pull request by jbampton**
We run mypy with pre-commit so it should be self contained

### CODE REVIEW

**Comment by BaseMax** [comment]
Also we can delete `types-PyYAML` from root, we need to have `types-PyYAML` inside `config` dir.

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 pre-commit
-mypy
 types-PyYAML
```
Comment (jbampton) [inline requirements.txt:2]:
```suggestion
```

**Comment by BaseMax** [comment]
And please update requirements.txt file in the config dir.

**Comment by BaseMax** [comment]
> requirements.txt in `config` and `portal` dir

@BDadmehr0 Maybe you can do it as next PR.

Many thanks

**Pull request merged**